### PR TITLE
Settings: add new card for managing plugins.

### LINKED ISSUE #10694: Settings: Add card for autoupdates in Security settings
Autoupdates are hard to find in wp-admin and link out to Calypso /plugins/manage to activate piecemeal. Let's make them easier to find in wp-admin and add a card for it in Security settings.

ref: p6TEKc-2m8-p2 (RB / item 5.19)

### PR DESCRIPTION
**Pull request by jeherve**
Fixes #10694 

#### Changes proposed in this Pull Request:

This PR adds a new card for managing plugins.

**When connected**
screenshot 2018-12-10 at 22 36 23

**When in dev mode**

screenshot 2018-12-10 at 22 49 55

#### Testing instructions:

* Go to Jetpack > Settings while your site is in dev mode, with Manage active, or with Manage off.
* Make sure the new card in the Security tab looks good.

#### Proposed changelog entry for your changes:
* Settings: add new card for plugins management.

### CODE REVIEW

**Comment by oskosk** [comment]
This one also needs a rebase to help 0c1819c9965b4de8699fe7c1842470f048606ef7

**Review by oskosk — DISMISSED** [review]
Good to merge after rebasing!

Note that manage being on or off (mentioned in the testing instructions) does not make a difference in the aspect of this new card.

**Comment by jeherve** [comment]
I rebased. Ready for a design review before merge, maybe?

**Comment by keoshi** [comment]
@jeherve From now on, it would be awesome if you could add the entire @Automattic/jetpack-design group on the PR review. I've asked Derek to add this yesterday, so everyone can consistently ping us and get a faster review hopefully. Will review right away — thanks!

**Review by oskosk — APPROVED** [review]

File: _inc/client/security/manage-plugins.jsx
Comment (oskosk) [inline _inc/client/security/manage-plugins.jsx]: Can we make `configLink` be a class method instead ?

**Review by oskosk — DISMISSED** [review]
Sorry, I want to propose something

File: _inc/client/security/manage-plugins.jsx
Comment (jeherve) [reply _inc/client/security/manage-plugins.jsx]: Done in e2bd16022cabef9f9f0c111c9c62093d4e3b44bc

**Review by keoshi — COMMENTED** [review]
Works great in my testing! One thing I'd like to address before we merge if possible — the corresponding card on the dashboard looks different:

image

- Dashboard card: replace chevron/arrow with external icon.
- Dashboard card: open link in new tab/window.
- Settings card: change label to “Manage your plugins”.

**Review by oskosk — APPROVED** [review]
LGTM!

**Pull request merged**
